barkibu-kb.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |spec|
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
-  spec.required_ruby_version = '>= 2.6'
+  spec.required_ruby_version = '>= 2.6', '< 3.3'
```

Comment:
Actually, maximum is any patch version of 3.2.
`< 3.3` ensures the gem runs below 3.3. I set that out of caution because we haven't tested the gem on Ruby 3.3.